web/modules/custom/dpl_react_apps/dpl_react_apps.routing.yml

```diff
@@ -18,3 +18,12 @@ dpl_react_apps.work:
     _controller: '\Drupal\dpl_react_apps\Controller\DplReactAppsController::work'
   requirements:
     _permission: 'access content'
+
+dpl_react_apps.reader:
+  path: '/reader'
```

Comment:
Please consider making the identifier and/or order id arguments to the path.

As far as I can tell they are required for the reader to work. We do the same for e.g. works